crates/relayer/src/chain/eth/event/monitor.rs

```diff
@@ -81,13 +84,26 @@ impl EthEventMonitor {
             }
         }
 
+        // process incoming initial checkpoint
+        if let Ok(checkpoint) = self.create_receiver.try_recv() {
```

Comment:
Only one message from `create_receiver` will be sent, and most of the time we won't receive it as we already have a base slot. Is it right?